css/video.css

```diff
@@ -0,0 +1,185 @@
+}
+.control div.paused{
+  background:url(../photos/control.png) no-repeat 0 -30px;
+}
+.control div.btnStop{
+  background:url(../photos/control.png) no-repeat 0 -60px;
+}
+.control div.spdText{
+  border:none;
+  font-size:14px;
+  line-height:30px;
+  font-style:italic;
+}
+.control div.selected{
+  font-size:15px;
+  color:#ccc;
+}
+.control div.sound{
+  background:url(../photos//control.png) no-repeat -88px -30px;
+  border:none;
+  float:right;
+}
+.control div.sound2{
+  background:url(../photos/control.png) no-repeat -88px -60px !important;
+}
+.control div.muted{
+  background:url(../photos/control.png) no-repeat -88px 0 !important;
+}
+.control div.btnFS{
+  background:url(../photos/control.png) no-repeat -44px 0;
+  float:right;
+}
+.control div.btnLight{
+  background:url(../photos/control.png) no-repeat -44px -60px;
+  border-left:1px solid #404040;
+  float:right;
+}
+.control div.lighton{
+  background:url(../photos/control.png) no-repeat -44px -30px !important;
+}
+/* Progress bar */
+.progress {
+  width:85%;
+  height:10px;
+  position:relative;
+  float:left;
+  cursor:pointer;
+  background: #444; /* fallback */
+  background:-moz-linear-gradient(top,#666,#333);
+  background:-webkit-linear-gradient(top,#666,#333);
+  background:-o-linear-gradient(top,#666,#333);
+  box-shadow:0 2px 3px #333 inset;
+  -moz-box-shadow:0 2px 3px #333 inset;
+  -webkit-box-shadow:0 2px 3px #333 inset;
+  border-radius:10px;
+  -moz-border-radius:10px;
+  -webkit-border-radius:10px;
+}
+.progress span {
+  height:100%;
+  position:absolute;
+  top:0;
+  left:0;
+  display:block;
+  border-radius:10px;
+  -moz-border-radius:10px;
+  -webkit-border-radius:10px;
+}
+.timeBar{
+  z-index:10;
+  width:0;
+  background: #3FB7FC; /* fallback */
+  background:-moz-linear-gradient(top,#A0DCFF 50%,#3FB7FC 50%,#16A9FF 100%);
+  background:-webkit-linear-gradient(top,#A0DCFF 50%,#3FB7FC 50%,#16A9FF 100%);
+  background:-o-linear-gradient(top,#A0DCFF 50%,#3FB7FC 50%,#16A9FF 100%);
+  box-shadow:0 0 1px #fff;
+  -moz-box-shadow:0 0 1px #fff;
+  -webkit-box-shadow:0 0 1px #fff;
+}
+.bufferBar{
+  z-index:5;
+  width:0;
+  background: #777;
+  background:-moz-linear-gradient(top,#999,#666);
+  background:-webkit-linear-gradient(top,#999,#666);
+  background:-o-linear-gradient(top,#999,#666);
+  box-shadow:2px 0 5px #333;
+  -moz-box-shadow:2px 0 5px #333;
+  -webkit-box-shadow:2px 0 5px #333;
+}
+/* time and duration */
+.time{
+  width:15%;
+  float:right;
+  text-align:center;
+  font-size:11px;
+  line-height:12px;
+}
+
+/* VOLUME BAR CSS */
+/* volume bar */
+.volume{
+  position:relative;
+  cursor:pointer;
+  width:70px;
+  height:10px;
+  float:right;
+  margin-top:10px;
+  margin-right:10px;
+}
+.volumeBar{
+  display:block;
+  height:100%;
+  position:absolute;
+  top:0;
+  left:0;
+  background-color:#eee;
+  z-index:10;
+}
+
+
+#init{
+  background:url(../photos/bigplay.png) no-repeat 50% 50% !important;
+  cursor:pointer;
```

Comment:
uh-oh, an !important. Not that I haven't been guilty of using that myself, but is there a way you can refactor to avoid using this?